packages/app_package_maker_exe/lib/src/make_exe_config.dart

```diff
@@ -10,6 +10,7 @@ class MakeExeConfig extends MakeConfig {
   String? publisherName;
   String? publisherUrl;
   bool? createDesktopIcon;
+  bool? addStartUp;
```

Comment:
Modifying `addStartup` to `launchAtStartup` would be better.

Reply:
fix https://github.com/leanflutter/flutter_distributor/pull/74/commits/5e9231cf624cb2b135a7b3e21cd5713d7a564b2e